Lib/test/test_typing.py

```diff
@@ -5191,6 +5191,18 @@ class B(Generic[T]):
         self.assertEqual(A[T], A[T])
         self.assertNotEqual(A[T], B[T])
 
+    def test_buggy_eq(self):
+        class BrokenEq(abc.ABCMeta):
```

Comment:
Doesn't need to be `ABCMeta` to trigger this; just `type` is sufficient:

```suggestion
        class BrokenEq(type):
```